quickstarts/android/02-customlogin.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Auth0 Android Quickstarts - 2. Custom Login
+description: This tutorial will show you how to use the Auth0 authentication api in your Android project to create a custom login screen.
+name: Android
+alias:
+  - Android
+  - Login
+  - Custom
+  - Headless
+  - Quickstart
+  - Mobile 
+language:
+  - Android
+hybrid: false
+image: /media/platforms/android.png
+tags:
+  - quickstart
+  - headless
+  - custom
+  - login
+alias:
+  - android
+seo_alias: android
```

Comment:
The only items needed here are `title` and `description`, the others should be gone, since they're going to be inferred from the `index.yml` as far as I know.